dashboard/test/ui/runner.rb

```diff
@@ -240,8 +242,7 @@ def run_tests(arguments)
 features_to_run = passed_features.empty? ? all_features : passed_features
 browser_features = $browsers.product features_to_run
 
-git_branch = `git rev-parse --abbrev-ref HEAD`.strip
-ENV['BATCH_NAME'] = "#{git_branch} | #{Time.now}"
+ENV['BATCH_NAME'] = "#{GIT_BRANCH} | #{Time.now}"
```

Comment:
A little cleanup on the side - we were getting the current branch twice, for no good reason.